Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/ProjectController.cs

```diff
@@ -44,356 +44,494 @@ public class ProjectController : Controller
         #endregion
 
         #region public Methods
+
         /**
-     * @api {get} api/Project/projects 
-     * @apiVersion 1.0.0
-     * @apiName Project
-     * @apiGroup Project
-     * @apiSuccessExample {json} Success-Response:
-     * HTTP/1.1 200 OK 
-     * {
-     *     "description":"Get List of Projects"
-     * }
-     */
+         * @api {get} api/project 
+         * @apiVersion 1.0.0
+         * @apiName GetProjectsAsync
+         * @apiGroup Project
+         * @apiParam {null} no parameter
+         * @apiSuccessExample {json} Success-Response:
+         * HTTP/1.1 200 OK 
+         * [
+          * {
+          *   "Name":"Slack",
+          *   "SlackChannelName":"SlackChannelName",
+          *   "IsActive":"True",
+          *   "TeamLeaderId":"34d1af3d-062f-4bcd-b6f9-b8fd5165e367",
+          *   "CreatedBy" : "Smith",
+          *   "CreatedDate" : "10/02/2016",
+          *   "UpdatedBy": "Smith",
+          *   "UpdatedDate" : "10/02/2016"
+          *   "TeamLeader": 
+          *   {
+          *         "Id":"34d1af3d-062f-4bcd-b6f9-b8fd5165e367",
+          *         "FirstName" : "John",
+          *         "Email" : "jone@promactinfo.com",
+          *         "LastName" : "Doe",
+          *         "SlackUserName" :"John",
+          *         "IsActive" : "True",
+          *         "JoiningDate" :"10-02-2016",
+          *         "NumberOfCasualLeave":0,
+          *         "NumberOfSickLeave":0,
+          *         "UniqueName":null,
+          *         "Role":null,
+          *         "Password":null,
```

Comment:
No sir I am not returning password but I have return UserAc object which has password field. so I have write password field as null.